glide.yaml

```diff
@@ -13,34 +13,21 @@ import:
   subpackages:
   - require
 - package: github.com/thriftrw/thriftrw-go
-  version: master
-  subpackages:
-  - ast
-  - compile
-  - envelope
-  - protocol
-  - wire
+  version: ~0.2.1
 - package: github.com/uber-go/atomic
   version: ^1.0.0
 - package: github.com/uber/tchannel-go
-  version: ^1.0.9
-  subpackages:
-  - thrift
+  version: ^1.2.0
 - package: golang.org/x/net
   version: master
   subpackages:
   - context
 - package: gopkg.in/yaml.v2
   version: master
+- package: github.com/opentracing/opentracing-go
+  version: ^1.0.0
 testImport:
 - package: github.com/apache/thrift
   version: master
-  subpackages:
-  - lib/go/thrift
 - package: github.com/yarpc/yarpc-go
-  version: master
-  subpackages:
-  - encoding/thrift
-  - transport
-  - transport/http
-  - transport/tchannel
+  version: ~0.2.1
```

Comment:
Yep only for testing, we do a full integration test with YARPC to make sure yab works against it. We should definitely update to the latest.